src/components/measureEditor/MeasureEditor.tsx

```diff
@@ -105,16 +105,15 @@ const MeasureEditor = () => {
   const [valuesetMsg, setValuesetMsg] = useState(null);
 
   const getValueSetErrors = async (
-    valuesetsArray: ElmValueSet[],
-    tgtValue: string
+    valuesetsArray: ElmValueSet[]
   ): Promise<ElmTranslationError[]> => {
     const valuesetsErrorArray: ElmTranslationError[] = [];
-    if (valuesetsArray && tgtValue) {
+    if (valuesetsArray) {
```

Comment:
I liked how the terminologyServiceApi wraps the Code validation call in a umlsLoggedIn check.

Either pass in the checkLogin value here similar to the call to validate Code, or check for it here and move the code validate check to this file as well.